flow/connectors/snowflake/snowflake.go

```diff
@@ -903,8 +905,18 @@ func generateCreateTableSQLForNormalizedTable(
 
 	// add a _peerdb_is_deleted column to the normalized table
 	// this is boolean default false, and is used to mark records as deleted
-	createTableSQLArray = append(createTableSQLArray,
-		fmt.Sprintf(`"%s" BOOLEAN DEFAULT FALSE,`, isDeletedColumnName))
+	if softDeleteColName != "" {
+		createTableSQLArray = append(createTableSQLArray,
+			fmt.Sprintf(`"%s" BOOLEAN DEFAULT FALSE,`, softDeleteColName))
+	}
+
+	// add a _peerdb_synced column to the normalized table
+	// this is a timestamp column that is used to mark records as synced
+	// default value is the current timestamp (snowflake)
+	if syncedAtColName != "" {
+		createTableSQLArray = append(createTableSQLArray,
+			fmt.Sprintf(`"%s" TIMESTAMP DEFAULT CURRENT_TIMESTAMP,`, syncedAtColName))
```

Comment:
could we use this default instead of explicitly setting in the merge statement?

Reply:
in merge statement we have to set it because when we update the column we have to update the timestamp.